### PR TITLE
Allow to pass descriptor into API functions

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @AlexanderMoskovkin @churkin @VasilyStrelyaev

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -52,14 +55,26 @@
...
+
+## *async* findWindow(pageTitle) ⇒ <code>object</code>
+Retrieves a platform-specific window descriptor for the window that contains a web page with the specified title.
+
+**Kind**: global [async](http://tc39.github.io/ecmascript-asyncawait/) function  
+**Returns**: <code>object</code> - a platform-specific window descriptor that can be used as a window identifier in some other API functions.  
+
+| Param | Type | Description |
+| --- | --- | --- |
+| pageTitle | <code>string</code> | The title of the web page opened in the window which descriptor should be retrieved. |
```
Comment (VasilyStrelyaev) [inline API.md]:
```
| pageTitle | <code>string</code> | The title of the web page opened in a window whose descriptor should be retrieved. |
```

File: API.md [hunk 2]
```diff
@@ -52,14 +55,26 @@
...
-| pageTitle | <code>string</code> | Specifies the title of the web page opened in the browser. |
+| windowId | <code>string</code> &#124; <code>object</code> | The title of the web page opened in the window or a descriptor returned by findWindow                                     of the window that should be closed. |
+
+<a name="findWindow"></a>
+
+## *async* findWindow(pageTitle) ⇒ <code>object</code>
+Retrieves a platform-specific window descriptor for the window that contains a web page with the specified title.
+
+**Kind**: global [async](http://tc39.github.io/ecmascript-asyncawait/) function  
+**Returns**: <code>object</code> - a platform-specific window descriptor that can be used as a window identifier in some other API functions.
```
Comment (VasilyStrelyaev) [inline API.md]:
```
**Returns**: <code>object</code> - a platform-specific window descriptor that can be used as a window identifier. 
```

File: API.md [hunk 3]
```diff
@@ -125,15 +140,15 @@ Gets the viewport size for the specified device.
...
+## *async* isMaximized(windowId) ⇒ <code>boolean</code>
 Checks if the specified browser window is maximized.
 
 **Kind**: global [async](http://tc39.github.io/ecmascript-asyncawait/) function  
 **Returns**: <code>boolean</code> - `true` if the browser window is maximized  
 
 | Param | Type | Description |
 | --- | --- | --- |
-| pageTitle | <code>string</code> | The title of the web page opened in the window to check if this window is maximized. |
+| windowId | <code>string</code> &#124; <code>object</code> | The title of the web page opened in the window or a descriptor returned by findWindow                                     to check if this window is maximized. |
```
Comment (VasilyStrelyaev) [inline API.md]:
```
| windowId | <code>string</code> &#124; <code>object</code> | The title of the web page opened in the window or a descriptor returned by findWindow. |
```

**Comment by AlexanderMoskovkin** [comment]
Please add id of issues that will be closed with this PR

**Comment by VasilyStrelyaev** [comment]
I thought it will be `windowDescriptor`

File: API.md [hunk 4]
```diff
@@ -1,9 +1,12 @@
 ## Functions
 
 <dl>
-<dt><a href="#close">async close(pageTitle)</a></dt>
+<dt><a href="#close">async close(windowId)</a></dt>
```
Comment (AlexanderMoskovkin) [inline API.md]: API is outdated. Should `windowDescriptor `

**Pull request merged**